test/functional/setup.ts

```diff
@@ -41,6 +45,12 @@ class FunctionalTestSetup {
   }
 
   public afterEach = () => {
+    for (const scraper of this.scrapers) {
+      if ((scraper as any).runtime.tools.store.database.open) {
```

Comment:
is there ever a case where these are closed before we get to the after block?

Reply:
Not since removing the `close()` call in `onComplete`, I can remove the if statement